subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c

```diff
@@ -169,12 +182,23 @@ static void lp_cu_ntf(struct ll_conn *conn, struct proc_ctx *ctx)
 
 static void lp_cu_complete(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t evt, void *param)
 {
-	if (!ntf_alloc_is_available()) {
-		ctx->state = LP_CU_STATE_WAIT_NTF;
-	} else {
-		lp_cu_ntf(conn, ctx);
+	/*
+	 * ConnUpd: Notify Host if any of the three conn. params have changed.
+	 * If no conn params are changed, the Host would not be notified.
+	 *
+	 * ParamReq: Notify Host.
+	 */
+	if ((ctx->proc == PROC_CONN_UPDATE) && (ctx->data.cu.params_changed == 0U)) {
```

Comment:
The (ctx->proc == PROC_CONN_UPDATE) && will be removed,
as shall always apply here.